src/lib/Libcmds/parse_depend.c

```diff
@@ -147,6 +147,10 @@ parse_depend_item(char *depend_list, char **rtn_list, int *rtn_size)
 			if (append_string(rtn_list, deptypes[i], rtn_size))
 				return 1;
 
+			/* It an error if there are no values after ':' */
+			if (*c == '\0')
```

Comment:
should we check for any trailing whitespaces after ':' ?

Reply:
Actually that job is done in the caller. It separates the dependency list on commas and then for each dependency it removes trailing spaces before calling parse_depend_item